src/textual/app.py

```diff
@@ -3023,10 +3029,14 @@ def _end_update(self) -> None:
     def _refresh_notifications(self) -> None:
         """Refresh the notifications on the current screen, if one is available."""
         # If we've got a screen to hand...
-        if self.screen is not None:
```

Comment:
I don't know why this didn't raise a typic error, as `self.screen` was not typed as returning None